cmd/transcriber/call/tracks.go

```diff
@@ -33,6 +34,7 @@ const (
 	trackInFrameSize          = trackAudioFrameSizeMs * trackInAudioRate / 1000  // The input frame size in samples
 	trackOutFrameSize         = trackAudioFrameSizeMs * trackOutAudioRate / 1000 // The output frame size in samples
 	audioGapThreshold         = time.Second                                      // The amount of time after which we detect a gap in the audio track.
+	rtpTSWrapAroundThreshold  = trackInAudioRate                                 // The threshold to detect if the RTP timestamp has wrapped around (one second worth of samples).
```

Comment:
This should be `trackInFrameSize` I think, and `(one frame worth of samples)`, no?

Reply:
aside for discussion, no necessarily a change: the naming of `trackInFrameSize` should maybe better be `trackInSamplesPerPacket` or something, because I think you can encode multiple frames per packet, right?

Reply:
This is purposely more than a single frame/packet to account for potential loss. Problem with wraparound is that if we don't spot it then we are indefinitely wrong so trying to be on the safe side and account for up to a second of data loss.